app/controllers/results_controller.rb

```diff
@@ -5,7 +5,23 @@ def index
     redirect_to new_result_path
   end
 
-  def new; end
+  def new
+    result_histories =
+      Result.select('DISTINCT ON (place_id) *').order(place_id: :desc).limit(5)
```

Comment:
`select('DISTINCT ON (place_id')` は、 `distinct(:place_id)` で書けますね。
あんまり生SQLを書かないほうが良いです。

```suggestion
      Result.distinct(:place_id).order(place_id: :desc).limit(5)
```

Reply:
コメントありがとうございます！
```
Result.distinct(:place_id).order(place_id: :desc).limit(5)
```
でやったところ、place_idが被った状態で検索履歴が表示されてしまい、`distinct(:place_id)`が効いていないようでした。
以下のissueに該当する問題ないのですがなぜ`distinct(:place_id)`が効かないのか調べても原因が分からない状況です。`distinct(:place_id)`が効かなくなる原因ご存知でしたらアドバイスいただけますと幸いです。
https://github.com/pekorinko/review_check/issues/26#issuecomment-824825348


【Result.distinct(:place_id).order(place_id: :desc).limit(5)　の場合】
- **place_idがかぶる**

![image](https://user-images.githubusercontent.com/61894818/116814668-09edfa80-ab95-11eb-8cda-15b94ec00a51.png)


【 Result.select('DISTINCT ON (place_id) *').order(place_id: :desc).limit(5)　の場合】
- **place_idが被らない**

![image](https://user-images.githubusercontent.com/61894818/116814621-d7440200-ab94-11eb-9e2c-5244f80cd1b3.png)